Backend/BackendAPI/Migrations/20210331173321_AddedLoginPassword.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace BackendAPI.Migrations
+{
+    public partial class AddedLoginPassword : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
+                name: "Login",
+                table: "Users",
+                maxLength: 20,
+                nullable: false,
+                defaultValue: "");
+
+            migrationBuilder.AddColumn<string>(
```

Comment:
Na ten sprint hasło w plaintextcie spoko. Okulewicz/Bródka na pewno będą mieć z tym problem później. NIe wyobrażasz sobie nawet ile takich kwiatków ląduje później na produkcji xD. Tępimy hasła w plain tekście, ale to w sprincie 2.